### PR TITLE
Refactor traces attributes

### PR DESCRIPTION
**Pull request by ka3de**
## What?

Rename traces attributes.

## Why?

 To better comply with conventions on attribute naming recommended by Open Telemetry. See [docs](https://opentelemetry.io/docs/specs/otel/common/attribute-naming/).

## Checklist

- [X] I have performed a self-review of my code
- [ ] I have added tests for my changes
- [ ] I have commented on my code, particularly in hard-to-understand areas

## Related PR(s)/Issue(s)

#1100

### CODE REVIEW

File: common/frame_session.go [hunk 1]
```diff
@@ -352,8 +352,11 @@ func (fs *FrameSession) parseAndEmitWebVitalMetric(object string) error {
 	})
 
 	_, span := TraceEvent(
-		fs.ctx, fs.targetID.String(), "web_vital", wv.SpanID,
-		trace.WithAttributes(attribute.Float64(wv.Name, value), attribute.String("rating", wv.Rating)))
+		fs.ctx, fs.targetID.String(), "web_vital", wv.SpanID, trace.WithAttributes(
+			attribute.String("web_vital.name", wv.Name),
```
Comment (ka3de) [inline common/frame_session.go:356]: Notice that there is a considerable change here, as previously we were using the Web Vital name itself as attribute key.

**Comment by ka3de** [comment]
I'm considering whether we should also change the "navigation span" [here](https://github.com/grafana/xk6-browser/blob/47c98bb9923c66264f69033fe6ae01ff0d0e7064/common/frame_session.go#L759) so the actual span name is `navigation` and the URL stays just as the attribute.
I have some doubts on this, as our initial traces/timeline representation is kind of "atypical" and we used this span to organize the rest. Let me know what are your thoughts.

**Review by ankur22 — APPROVED** [review]
LGTM

File: trace/trace.go [hunk 2]
```diff
@@ -12,7 +12,7 @@ import (
 	k6lib "go.k6.io/k6/lib"
 )
 
-const tracerName = "browser"
+const tracerName = "k6.browser"
```
[the remark this answers is not in the document]
Comment (ka3de) [reply trace/trace.go:15]: Yes, to give more context to this specific tracer, as we do not know for certain the data that will be stored in the same backend to which the traces are sent to.

**Review by inancgumus — APPROVED** [review]

**Comment by ka3de** [comment]
> I'm considering whether we should also change the "navigation span" [here](https://github.com/grafana/xk6-browser/blob/47c98bb9923c66264f69033fe6ae01ff0d0e7064/common/frame_session.go#L759) so the actual span name is navigation and the URL stays just as the attribute.

This will be done in a separate PR. See #1141

**Pull request merged**